ogc/src/main/scala/geotrellis/server/ogc/style/ColorRampStyle.scala

```diff
@@ -59,21 +59,22 @@ case class ColorRampStyle(
     // The number of stops between each provided break
     val numStops: Int = stops.getOrElse(colorRamp.colors.length)
 
-    // The colors, interpolated (invisible added for values below our minimum break)
-    val interpolatedColors: Vector[Int] =
-      0x00000000 +: colorRamp.stops(numStops)
+    // The colors, interpolated (colors added at start and end for out of bounds values)
+    val minColor = if (clampWithColor) colorRamp.colors.head else 0x00000000
+    val maxColor = if (clampWithColor) colorRamp.colors.last else 0x00000000
+    val interpolatedColors: Vector[Int] = minColor +: colorRamp.stops(numStops) :+ maxColor
 
     val interpolatedBreaks: Array[Double] =
-      breaks(hists, interpolatedColors.length)
+      breaks(hists, interpolatedColors.length) :+ Double.MaxValue
```

Comment:
Hm, why does the `Double.MaxValue` in the interpolation breaks?

Reply:
I _think_ if I'm remembering correctly that without this colors above the max-render value would always render transparent.

Reply:
Verified. If you want to render a color for each bin, you need a bin that represents everything greater than the max-render value. If I remove this extra appended value then values between max-render and Double.MaxValue aren't represented by a bin and are therefore always rendered as transparent pixels.

Reply:
I mention this only for the sake of airing the knowledge and think your solution is probably nicer.

An alternate way to control rendering outside of the binds provided within a `ColorMap` is to initialize the `ColorMap` with a custom `ColorMap.Options` object:
```
val breaks = ???
val ramp = ???
val fallbackRGB = ???
val options = ColorMap.Options.DEFAULT.copy(fallbackColor = fallbackRGB)
ramp.toColorMap(breaks, ramp, ColorMap
```